h2o-core/src/main/java/water/fvec/FileVec.java

```diff
@@ -194,7 +194,7 @@ public static int calcOptimalChunkSize(long totalSize, int numCols, long maxLine
     }
     else {
       // New Heuristic
-      int minNumberRows = 10; // need at least 10 rows (lines) per chunk (core)
+      int minNumberRows = 200; // need at least 200 rows (lines) per chunk (core) - that's needed for files with headers
```

Comment:
I thought that can. But there are cases where chunks go in parallel, like in ParserTestARFF.testNumSplit() - there seems to be no way to figure out when chunks number 0 come in parallel.